examples/mediatek/model_export_scripts/llama.py

```diff
@@ -0,0 +1,184 @@
+import os
+import sys
+if os.getcwd() not in sys.path:
+    sys.path.append(os.getcwd())
+import torch
+import argparse
+import struct
+from tqdm import tqdm
+from torch.ao.quantization.quantize_pt2e import convert_pt2e, prepare_pt2e
+from executorch.backends.mediatek.neuropilot import NeuropilotPartitioner, NeuropilotQuantizer, Precision
+from executorch.exir.backend.backend_details import CompileSpec
+from executorch import exir
+from datasets import load_dataset
+from aot_utils.llm_utils.sanity_checks import *
+from aot_utils.llm_utils.utils import (
+    resolve_model_classes,
+    get_normalized_config,
+    generate_mask,
+    get_dest_path,
+    get_dirname,
+    get_embedding_layer,
+    get_exp_name,
+    get_export_shapes,
+    get_master_rot_emb,
+    load_checkpoints,
+)
+from aot_utils.llm_utils.preformatter import Preformatter
+import warnings
+warnings.filterwarnings('ignore')
+
+def get_argument_parser():
+    parser = argparse.ArgumentParser(
+        description='Run Export to ET for suppoorted LLM models.', allow_abbrev=False)
+    parser.add_argument('config', type=str,
+        help="[Required] Model config json file. "
+        "Model config must be in same directory as all model weight bins and tokenizer files.")
+    parser.add_argument('-p', '--precision', type=str, default='A16W8', choices=['A16W4', 'A16W8', 'A16W16', 'A8W4', 'A8W8'],
+        help="Precision to quantize entire model to.")
+    parser.add_argument('-n', '--num_chunks', type=int, default=4,
```

Comment:
If it's stories model (the same model arch as llama2, 125M parameters), is the flow going to work? I believe in this case, we don't need to chunk the model: 
model checkpoint: https://huggingface.co/karpathy/tinyllamas/resolve/main/stories110M.pt
tokenizer: https://raw.githubusercontent.com/karpathy/llama2.c/master/tokenizer.model

Reply:
Hi @cccclai , generally, if its <= 3B param models we can do 1 chunk (ie effectively no chunking). 

Users just need to set -n 1 in the export shell script. 

For >= 7B params we need minimum 2 chunks.